policy/modules/contrib/sandboxX.te

```diff
@@ -52,6 +52,10 @@ allow sandbox_xserver_t self:process setsched;
 allow sandbox_xserver_t self:shm create_shm_perms;
 allow sandbox_xserver_t self:tcp_socket create_stream_socket_perms;
 
+fs_getattr_xattr_fs(sandbox_xserver_t)
+allow sandbox_xserver_t sandbox_xserver_tmpfs_t:file map;
+read_files_pattern(sandbox_xserver_t, sysctl_dev_t, sysctl_dev_t0)
```

Comment:
Please use 'kernel_read_device_sysctls(sandbox_xserver_t)'. You can make use of `audit2allow -R` for a hint where to look for the right interface, not always to find the most appropriate one.

Also keep the standard order - allow rules, files/patterns, kernel, base interfaces, modules in optional block, orderer alphabetically - if possible - inside.

Reply:
Well, also together with #L55 move a few lines below. #L56 should also go down close to `manage_dirs_pattern(sandbox_xserver_t, sandbox_xserver_tmpfs_t, sandbox_xserver_tmpfs_t)`

Reply:
I can't see any changes to the previous state. Did you push them?

Reply:
Good direction, just please move #L60 7 lines down, #L70 2 down and #L69 about 30 down, too, so they are grouped well for better reading by others.